sphinx/tutorial/rose/furthertopics/date-time-manipulation.rst

```diff
@@ -21,13 +21,21 @@ directory.
 The ``rose date`` Command
 -------------------------
 
+.. _isodatetime: https://github.com/metomi/isodatetime
+
+.. note::
+
+   ``rose date`` has been superseded by `isodatetime`_ which offers a
+   compatible CLI interface with more features (just swap out ``rose date``
+   for ``isodatetime``), as well as a Python API.
+
```

Comment:
Rather than documenting this deprecated command, how about `s/rose date/isodatetime/g`?

Reply:
Just had a look at doing this. But it turns out `rose date` accepts an option `-c` whereas `isodatetime` does note (even though it is mentioned in the help text)